### ISSUE TITLE
Additional params for siteshooter.yml

### ISSUE THREAD

**Issue by benhalverson** [devopsgroup-io/siteshooter#63]
Right now the company I am at is extending the cli app. 
Our use case is this.
Currently a PM goes to a website and manually screenshots every page/state for a website, webapp or iPad app. 
I suggested we make a UI on top of siteshooter. 
One limitation right now are the more times we run siteshooter the longer the `sitemap.xml` file gets after going through each site.
We have added a 2 new parameters in the siteshooter.yml 
```
sitemapPath: 'websitename-sitemap.xml'
injectjsPath: 'websitename-inject.js'
```

Eventually I will create a RESTful api that will work with a UI that I will also build
@stevebritton Have you done anything like this yet for your own projects?

**Comment by stevebritton** [comment]
Hey @benhalverson,

I just released a new version of siteshooter (v1.12.4) which includes a new argument to specify a working directory. With this argument, you'll be able to run different **siteshooter.yml** files without having to change your working directory and then run siteshooter.

**Example Directory Structure:**

```
var/
|—— www/
|   |—— sites/
|   |   |—— product1/
|   |   |   |—— siteshooter.yml
|   |   |—— product2/
|   |   |   |—— siteshooter.yml
```
**Example Setting Working Directory:**
```
siteshooter --cwd=www/sites/product1/ 
```
Regarding a RESTful API, we're actually in the process of developing a full-blown dashboard, so I'll be sure to ping you once we're ready to start doing some BETA testing.

-Steve

**Comment by benhalverson** [comment]
awesome thanks for the update @stevebritton
